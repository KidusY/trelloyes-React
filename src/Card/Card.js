import React from 'react';
import './Card.css'

function Card(props) {
    return (
        <div className="Card">
            <button type="button" onClick={()=>props.onDelete(props.card.id,props.id)}>Delete</button>
            <h3>{props.card.title}</h3>
            <p>{props.card.content}</p>
        </div>
    )
}

export default Card;