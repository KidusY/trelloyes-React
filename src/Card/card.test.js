import React from 'react'
import ReactDOM from 'react-dom'
import Card from './Card'
import renderer from 'react-test-renderer'

it ('I just wanted to do the smoke test', () => {
    const div = document.createElement('div')
    ReactDOM.render(<Card />, div)
    ReactDOM.unmountComponentAtNode(div)
})

it ('I just wanted to do the snapshot test', () => {
    const myRenderedElement1= renderer.create(<Card />).toJSON();
    expect(myRenderedElement1).toMatchSnapshot();
})
