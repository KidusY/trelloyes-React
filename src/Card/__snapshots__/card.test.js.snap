// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`I just wanted to do the snapshot test 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3 />
  <p />
  <p />
</div>
`;
